From 932dc7659d836a6cb5feb9239c8578a553f22a92 Mon Sep 17 00:00:00 2001
From: neomilium <neomilium@gmail.com>
Date: Sat, 13 Mar 2010 00:53:18 +0100
Subject: [PATCH 8/8] Add "Le Grand Journal" to supported vod.

---
 canal.py |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/canal.py b/canal.py
index 1493c9f..5e9fb15 100755
--- a/canal.py
+++ b/canal.py
@@ -41,6 +41,7 @@ available_show = {"A": "discrete",
                   "J": "petit", 
                   "L": "petite", 
                   "M": "meteo", 
+                  "Q": "grand journal", 
                   "P": "pepites", 
                   "R": "groland", 
                   "S": "sav", 
@@ -54,6 +55,7 @@ nb_videos_show = {"discrete": 1,
                   "petit": 3,
                   "petite": 1, 
                   "meteo": 1, 
+                  "grand journal": 5, 
                   "pepites": 1, 
                   "groland": 1, 
                   "sav": 1, 
-- 
1.6.3.3

